ste/xfer-anyToRemote-file.go

```diff
@@ -161,6 +163,22 @@ func ValidateTier(jptm IJobPartTransferMgr, blobTier *blob.AccessTier, client IB
 	}
 }
 
+func DeleteDstBlob(jptm IJobPartTransferMgr, client *blockblob.Client) {
```

Comment:
Is there a reason this code needs to live in xfer-anyToRemote-file even though its specific to block blob senders?

Reply:
We can move this code to blob-sender Prologue(), so that it is done exactly once per blob.

Reply:
yes, each blob sender should have a sync.Once but I disagree that it should be in Prologue cause we only want to do it once we get a invalid blob or block

Reply:
So, when we stage the blocks, the 'chunks' are out of order so if we try to delete the destination blob after getting the error 'InvalidBlockOrBlob', some blocks successfully "stage" and it prematurely deletes the blob causing it to fail. 

Agree with Narasimha. I tested some more and moving the code to Prologue() ensures that if there are uncommitted blocks on a destination blob, they are deleted before we start the chunking process.

Reply:
Updated!